templates/customer/password-new.tpl

```diff
@@ -48,13 +48,13 @@
         <label class="form-label">{l s='Confirmation' d='Shop.Forms.Labels'}</label>
         <div class="input-group password-field js-parent-focus">
         <input class="form-control js-child-focus js-visible-password" type="password" data-validate="isPasswd" name="confirmation" value="">
-            <button
-              class="btn btn-primary"
-              type="button"
-              data-action="show-password"
-              data-text-show="{l s='Show' d='Shop.Theme.Actions'}"
-              data-text-hide="{l s='Hide' d='Shop.Theme.Actions'}"
-            >
+          <button
+            class="btn btn-primary"
+            type="button"
+            data-action="show-password"
+            data-text-show="{l s='Show' d='Shop.Theme.Actions'}"
+            data-text-hide="{l s='Hide' d='Shop.Theme.Actions'}"
+          >
             <i class="material-icons">visibility</i>
```

Comment:
Is there a reason this one for not adding `aria-hidden` on this one ?

Reply:
@matthieu-rolland It's because the button does not have any text, so the icon actually serves a purpose.

Reply:
@matthieu-rolland Fixed on PR #508